examples/tracing/kernel_cache_alloc_free.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python
+#
+# kernel_cache_alloc_free.py	Kernel memory (alloc-free) monitoring tool.
+#		
+#
+# Written as a basic tool example of using ePBF to track of kernel cache allocation and deallocation process.
```

Comment:
The same here and below, the line is too long, could you break it into multiple lines. Although sometimes we do have exceptions, it would be good if you can constrain each line to maximum 80 characters.